backend/src/email/routes/email-transactional.routes.ts

```diff
@@ -236,5 +236,243 @@ export const InitEmailTransactionalRoute = (
+   *     get:
+   *       security:
+   *         - bearerAuth: []
+   *         - cookieAuth: []
+   *       tags:
+   *         - Email
+   *       summary: "List transactional emails"
+   *       parameters:
+   *         - in: query
+   *           name: limit
+   *           description: max number of messages returned
+   *           required: false
+   *           schema:
+   *             type: integer
+   *             minimum: 1
+   *             maximum: 100
+   *             default: 10
+   *         - in: query
+   *           name: offset
+   *           description: offset to begin returning messages from
+   *           required: false
+   *           schema:
+   *             type: integer
+   *             minimum: 0
+   *             default: 0
+   *         - in: query
+   *           name: status
+   *           description: status of messages to filter for
+   *           required: false
+   *           schema:
+   *             type: array
+   *             items:
+   *               type: string
+   *               enum: [UNSENT, ACCEPTED, SENT, BOUNCED, DELIVERED, OPENED, COMPLAINT]
+   *           style: form
+   *           explode: true
+   *         - in: query
+   *           name: created_at
+   *           description: >
+   *              Filter for created_at timestamp of messages:
+   *                - gt: greater than
+   *                - gte: greater than or equal
+   *                - lt: less than
+   *                - lte: less than or equal
+   *           required: false
+   *           schema:
+   *             type: object
+   *             minProperties: 1
+   *             properties:
+   *               gt:
+   *                 type: string
+   *                 format: date-time
+   *               gte:
+   *                 type: string
+   *                 format: date-time
+   *               lt:
+   *                 type: string
+   *                 format: date-time
+   *               lte:
+   *                 type: string
+   *                 format: date-time
+   *         - in: query
+   *           name: sort_by
+   *           description: >
+   *             Array of fields to sort by, default order is desc, but can be configured by adding a prefix of:
+   *              - plus sign (+) for ascending order
+   *              - minus sign (-) for descending order
+   *           required: false
+   *           schema:
+   *             type: array
+   *             default: [created_at]
+   *             items:
+   *               type: string
+   *               enum: [created_at, +created_at, -created_at, updated_at, -updated_at, +updated_at]
+   *
+   *       responses:
+   *         200:
+   *           description: Succcessfully retrieve a list of messages
+   *           content:
+   *             application/json:
+   *               schema:
+   *                 type: object
+   *                 required:
+   *                   - has_more
```

Comment:
```suggestion
   *                   - next_id
```

Perhaps provide a `next_id` instead, so that API users can rely on our API responses to paginate without having to do math on their end (i.e. `next_id` = `curr_id` + `offset`)

Reply:
We're doing `offset + limit` instead of a `start_id + limit` (or `end_id + limit`) though so giving `next_id` is not really helpful in this case

Reply:
Ah I meant `next_id` as a `next_offset_value`. But this isn't a big deal. I'm ok with `has_more`.